### PR TITLE
Enabling NuGetPackageVerifier

### PR DESCRIPTION
**Pull request by ajaybhargavb**
@Eilon

### CODE REVIEW

File: NuGetPackageVerifier.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        "rules": [
+            "AssemblyHasDocumentFileRule",
+            "AssemblyHasVersionAttributesRule",
+            "AssemblyHasServicingAttributeRule",
+            "AssemblyHasNeutralResourcesLanguageAttributeRule",
+            "SatellitePackageRule",
+            "StrictSemanticVersionValidationRule"
+        ],
+        "packages": {
+            "CultureInfoGenerator": { },
```
Comment (Eilon) [inline NuGetPackageVerifier.json]: I think this one is non-shipping, no?

Comment (Eilon) [inline NuGetPackageVerifier.json]: And if it is, can remove the change to the assembly info for that package.

Comment (ajaybhargavb) [inline NuGetPackageVerifier.json]: yes, will update.

**Comment by Eilon** [comment]
:shipit:

**Pull request merged**